### PR TITLE
Fix a possible crash from the mediaplayer impl NEWSDEVICE-2550

### PR DESCRIPTION
**Pull request by johywang**

### CODE REVIEW

File: library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java [hunk 1]
```diff
@@ -1112,7 +1112,13 @@ public long getCurrentPosition() {
         }
 
         if (!useExo) {
-            return positionOffset + videoView.getCurrentPosition();
+			try {
+				// sometimes the mediaplayer throws an IllegalStateException
+				return positionOffset + videoView.getCurrentPosition();
+			} catch (IllegalStateException i) {
+				i.printStackTrace();
```
Comment (jblebrun) [inline library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java]: No `printStackTrace` right? Maybe a `reportNonFatal`?

Comment (johywang) [inline library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java]: This is in the Exomedia fork. No crashlytics to be found here.

Comment (jblebrun) [inline library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java]: ohhh.

Comment (jblebrun) [inline library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java]: Is it Android-specific, so we could at least use Log.w() or whatever?

Comment (johywang) [inline library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java]: Yeah sure

**Pull request merged**